google/cloud/bigtable/table.h

```diff
@@ -40,12 +40,12 @@ namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
 /// The branch taken by a Table::CheckAndMutateRow operation.
 enum class MutationBranch {
+  /// The predicate provided to CheckAndMutateRow did not match and the
+  /// corresponding mutations (if any) were applied.
```

Comment:
Should that be "not applied"?

Reply:
There are two sets of mutations, one applied if the predicate matches, another applied when the predicate does not match. So "corresponding mutations" is technically correct.  But maybe I should be more explicit?

Reply:
I see.  I'm just naive, so "corresponding" was lost on me.  So yes, perhaps you could say `true_mutations` and `false_mutations` explicitly instead.